src/browser/SplashScreenProxy.js

```diff
@@ -138,13 +138,11 @@ function readPreferencesFromCfg(cfg) {
  * Shows and hides splashscreen if it is enabled, with a delay according the current preferences.
  */
 function showAndHide() {
```

Comment:
Is this possibly used somewhere else?
Or could this be used externally so someone could depend on the broken behavior?

Reply:
Its certainly possible that someone externally uses this and depends on the broken behavior. The only way to avoid that for certain would be to keep the if in this function and then also check for enabled in the caller when autohide is disabled.